scrapers/py_importers/py_common/utils.py

```diff
@@ -27,14 +27,20 @@
 ###########################
 ### GLOBAL VAR AND CONS ###
 ###########################
-DEBUG = True                    # enable if you want a more detailed log, beta folder and other useful things 
```

Comment:
A lot of these changes look like formatting. Since this is a common file used by also other scrapers you implement, can you confirm this will still work for them (in case some changes a part form the formatting is actually introduced?)

Reply:
I didn't removed it, just changed the line where it is (it is not my fault but autoformatter).